### ISSUE TITLE
TIMEOUT issue after groovy lint

### ISSUE THREAD

**Issue by YuquanCui** [nvuillam/npm-groovy-lint#434]
Does npm-groovy-lint request for any other IP or service? Because I case accross an timeout issue after lint running.
```text
Unexpected error: connect ETIMEDOUT 44.239.248.123:443
Error: connect ETIMEDOUT 44.239.248.123:443
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1138:16)
Error: Process completed with exit code 1.
```
